pyproject.toml

```diff
@@ -191,8 +191,8 @@ addopts = "-vs --cov-report term-missing --cov-report xml --dist loadscope --jun
 junit_duration_report = "call"
 
 [tool.pytest_env]
-PREFECT_LOGGING_LEVEL = "CRITICAL"
-INFRAHUB_LOG_LEVEL = "CRITICAL"
+PREFECT_LOGGING_LEVEL = "ERROR"
```

Comment:
it would be good if we could get it to work with `-vvv` this way we don't have much logs by default and we can increase the log level as needed

Reply:
I added changes to control testing logs using `pytest --log-level` so we can have a fine-grained control of logging level. Also, I have setup default logging to `WARNING`.